sammo.py

```diff
@@ -6,6 +6,7 @@
 from .src.gui.session import SammoActionSession
 from .src.gui.on_off_effort import SammoActionOnOffEffort
 from .src.core.session import SammoSession
+from .src.gui.add_observation_btn import AddObservationBtn
```

Comment:
I'd like to be consistent regarding the naming. So maybe `SammoAddObsverationBtn`?